### ISSUE TITLE
Exclude pages from history with regular exception

### ISSUE THREAD

**Issue by ThePrincelle** [mikecao/umami#1033]
Hi, 

First of all, thanks for this tool, we just migrated to Umami from Google Analytics and it's just perfect.

I was wondering if it was possible to exclude pages or aggregate them in he history to avoid paths like : 
`/guest/<id>` or `/view/<id>` to be counted out individually but as one page. 

I know that there is an option in the tracker configuration to disable the auto-tracking of pages. But my applications are quite large and I would like to avoid to specify for each page whether to enable tracking or not.

Thanks for your response,
Maxime

**Comment by mikecao** [comment]
I don't think we can exclude pages since they are actual pageviews. An upcoming feature is to be able to report and query on UTM query parameters. Maybe we can add URL aggregation to that.

Otherwise, your only option is to do it programmatically after disabling auto-tracking. You would send a modified path in the `umami.trackView` method. See https://umami.is/docs/tracker-functions

Another thing you can do is update the data directly through SQL. Find all instances of the url you want to modify and strip out the rest of the url.

**Comment by d19dotca** [comment]
Interesting. I have a similar use-case where I'd like to amalgamate/aggregate the count of /checkout/order-received/* instead of having 1 view each for /checkout/order-received/10002 and increasing every number with each order. It's cluttering the page view having so many of the individual order pages in the list of page views.
